### ISSUE TITLE
Issue installing when make on php5

### ISSUE THREAD

**Issue by roj4s** [piersharding/php-sapnwrfc#20]
After phpize, configure i put ./make and it returns all the error listed below:

sapnwrfc.c:2576:28: error: macro "RETURN_STRINGL" passed 3 arguments, but takes just 2
sapnwrfc.c:2192:88: error: macro "add_assoc_string" passed 4 arguments, but takes just 3
sapnwrfc.c:2607:28: error: macro "RETURN_STRINGL" passed 3 arguments, but takes just 2

and much much more ...

**Comment by piersharding** [comment]
Hi -

What version of PHP5 are you using ?

Cheers.

**Comment by roj4s** [comment]
Php 5.6 ubuntu 16.04

**Comment by piersharding** [comment]
Hi -

you might want to have a look at https://github.com/gkralik/php7-sapnwrfc .

Cheers.

**Comment by roj4s** [comment]
Yeah thanks I'm using that one ... However need a version compatible with php5.
